src/App.jsx

```diff
@@ -1,15 +1,64 @@
 import { hot } from 'react-hot-loader';
-import React from 'react';
+import React, { useState } from 'react';
+import { Authorize } from 'react-auth0-components';
 import { ThemeProvider } from '@material-ui/styles';
+import AuthContext from './utils/AuthContext';
+import { USER_SESSION } from './utils/constants';
 import theme from './theme';
 import Main from './Main';
 
-const App = () => (
-  <div>
-    <ThemeProvider theme={theme}>
-      <Main />
-    </ThemeProvider>
-  </div>
-);
+const App = () => {
+  const [authorize, setAuthorize] = useState(false);
+  const [authContext, setAuthContext] = useState({
+    authorize: () => setAuthorize(true),
+    unauthorize: () => {
+      setAuthorize(false);
+      setAuthContext({
+        ...authContext,
+        user: null,
+      });
+    },
+    user: null,
+  });
+
+  return (
+    <AuthContext.Provider value={authContext}>
+      <ThemeProvider theme={theme}>
+        <Authorize
+          authorize={authorize}
+          popup
+          domain={process.env.AUTH0_DOMAIN}
+          clientID={process.env.AUTH0_CLIENT_ID}
+          audience={process.env.AUTH0_AUDIENCE}
+          redirectUri={process.env.AUTH0_REDIRECT_URI}
+          responseType={process.env.AUTH0_RESPONSE_TYPE}
+          scope={process.env.AUTH0_SCOPE}
+          render={() => {
+            const session = localStorage.getItem(USER_SESSION);
```

Comment:
It looks like this code only runs when I click "Login". Is it possible to have a background login happen, if I've signed in previously? It's rather annoying to need to click "Login" every time I load the site.